android/shared/src/commonMain/kotlin/com/yeobaek/feature/nickname/NicknameViewModel.kt

```
@@ -0,0 +1,69 @@
+        )
+
+        viewModelScope.launch {
+            try {
+                userRepository.setUserData(uiState.nicknameValue)
+                uiState = uiState.copy(
+                    isEnabled = true,
+                    successNicknameSet = true,
+                )
+            } catch (e: CancellationException) {
+                throw e
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)